applications/DEMApplication/custom_strategies/strategies/explicit_solver_continuum.cpp

```diff
@@ -399,65 +399,125 @@ namespace Kratos {
     } //CreateContactElements
 
     void ContinuumExplicitSolverStrategy::SetCoordinationNumber(ModelPart& r_model_part) {
+
         ProcessInfo& r_process_info = r_model_part.GetProcessInfo();
 
-        const double in_coordination_number = r_process_info[COORDINATION_NUMBER];
+        const double desired_coordination_number = r_process_info[COORDINATION_NUMBER];
         double standard_dev = 0;
-        double out_coordination_number = ComputeCoordinationNumber(standard_dev);
+        double current_coordination_number = ComputeCoordinationNumber(standard_dev);
         int iteration = 0;
         int maxiteration = 100;
         double& added_search_distance = r_process_info[SEARCH_RADIUS_INCREMENT];
+        const bool local_coordination_option = r_process_info[LOCAL_COORDINATION_NUMBER_OPTION];
+        const bool global_coordination_option = r_process_info[GLOBAL_COORDINATION_NUMBER_OPTION];
         added_search_distance = 0.0;
         double amplification = 1.0;
 
         if(r_model_part.GetCommunicator().MyPID() == 0) {
```

Comment:
this check is no longer necessary, the logger only prints only on one rank, unless you use the `...ON_ALL_RANKS` logger macro

Reply:
ok, thanks